tests/scripts/minikube.sh

```diff
@@ -16,6 +16,85 @@ EOF
     minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
 }
 
+# workaround for kube-dns CrashLoopBackOff issue with RBAC enabled
+#issue https://github.com/kubernetes/minikube/issues/1734 and https://github.com/kubernetes/minikube/issues/1722
+enable_roles_for_RBAC() {
+ cat <<EOF | kubectl create -f -
```

Comment:
The indent from the other lines seems to be 4 spaces. Please change it here to look like the other lines.

Reply:
oops fixed now.